core/src/main/java/hudson/PluginManager.java

```diff
@@ -959,40 +959,40 @@ protected boolean filter(Method e) {
     }
 
     @Restricted(NoExternalUse.class)
-    public synchronized void resolveDependantPlugins() {
```

Comment:
While this is "public", it's "NoExternalUse" and thus I'm comfortable not worrying about an API break

Reply:
Disagree, we change `@Restricted` stuff for all kinds of reasons and it's the caller's problem.

Reply:
I agree with @daniel-beck, we can consider `@Restricted` methods as `@Beta`: can break anytime, you were not really supposed to use it anyway.